packages/react-data-grid/src/RowGroup.js

```diff
@@ -67,7 +72,8 @@ RowGroup.propTypes = {
   isExpanded: PropTypes.bool.isRequired,
   treeDepth: PropTypes.number.isRequired,
   name: PropTypes.string.isRequired,
-  renderer: PropTypes.func
+  renderer: PropTypes.func,
+  eventBus: PropTypes.isRequired
```

Comment:
is this valid? what's the shape of event bus?

Reply:
No, this should be an object or an object with `{ dispatch, subscribe }` methods but I am just using `object` for now